### ISSUE TITLE
App not working on Windows 11 Pro 21H2 22000.469 build

### ISSUE THREAD

**Issue by krp555** [QL-Win/QuickLook#1075]
Installed the QuickLook Latest version but was not able to see any file preview after selecting the file and then pressing the Space Bar key. Nothing happens and no error messages...

**Comment by rabelux** [comment]
Did you start QuickLook? Is there an icon in the tray?

**Comment by krp555** [comment]
Thanks for the quick reply.

Yes, QuickLook is started and I can see an icon of running QuickLook in the tray…

I tried QuickLook by starting with Administrative access. I have tried repair and reset options but still, nothing happens. Not able to see even image previews.

**Comment by rabelux** [comment]
There is a reason that there's a template for reporting issues... I'm *guessing* you're using the msi installer? Are you using a different explorer or trying to preview a file on your desktop?

**Comment by rabelux** [comment]
Pls fill out the missing information from the template and provide a screenshot.

**Comment by krp555** [comment]
Initially, I installed it from Microsoft Store and then also tried using MSI installer but it was not working. But as you suspected the reason is I was using another File Explorer from Microsoft Store – “Files” which is not supported by QuickLoop. Even I checked it with others File Explorer and QuickLook do not support them. Hope soon QuickLook will support other File Explorers also.

Now I found that QuickLook is working fine with windows native File Explorer and I can see all supported file types in the preview. While using locked pdf files it also alarms – failed to preview and report it to QuickLook’s author.

Thanks for the quick support in resolving the issue.

 

Regards

 

Krishnachandra Pardeshi
